src/containers/MessageTemplate/MessageTemplate.tsx

```diff
@@ -64,15 +68,17 @@ export const MessageTemplate: React.SFC<TemplateProps> = (props) => {
         query: FILTER_TEMPLATES,
         variables: queryVariables,
       });
-      cache.writeQuery({
-        query: FILTER_TEMPLATES,
-        variables: queryVariables,
-        data: {
-          sessionTemplates: templates.sessionTemplates.concat(
-            createSessionTemplate.sessionTemplate
-          ),
-        },
-      });
+
+      if (createSessionTemplate)
```

Comment:
@rathorevaibhav Can you please update this as per our discussion

Reply:
sorry I thought I had pushed. done now. please check.